pkgs/applications/graphics/upscayl/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}: let
+  pname = "upscayl";
+  version = "2.5.5";
+  name = "${pname}-${version}";
```

Comment:
You should not create the `name` attribute yourself, see for example: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/museeks/default.nix

There's no mention of `name` anywhere, only `pname`.

Reply:
Guess that works too!

Reply:
Because the `name` attribute will be generated by another layer if it doesn't exist. Therefore, there's no need to create it here.